nrk-tv-downloader.sh

```diff
@@ -370,16 +370,17 @@ function program_all()
     local season=$SEASON
     local html=$(curl $CURL_ "$url")
     local program_id=$(gethtmlMeta "$html" 'og:url' \
-        | grep -o -P "([A-Z]{4}[0-9]{8})"
+        | sed -E 's/.*([A-Z]{4}[0-9]{8}).*/\1/'
     )
 
     local seasons=$(gethtmlAttr "$html" "data-season" "data-season")
     if $season ; then
         seasons=$(gethtmlAttr "$html" "seasonid")
     fi
     local series_name=$(gethtmlMeta "$html" 'og:url' \
-        | grep -o -P "/serie/\K[^/]+"
+        | sed -E 's/.*serie\/([a-zA-Z0-9]*)\/.*/\1/'
```

Comment:
I am not sure what characters can be included in the `series name`, but a lot of them have been using `-` (dash).

So having the matching group like `[a-zA-Z0-9-]` or even just `[^/]`, so we are sure we are matching the whole name. Also use `+` for repetition after the matching group, as it will be more expressive that we don't want an empty match.